README.md

```diff
@@ -87,7 +87,7 @@ application.register('modal', Modal)
 ```
 
 ```html
-<div data-controller="modal">
+<div data-controller="modal" data-action="keydown@window->modal#closeWithKeyboard">
```

Comment:
If I may I don't think the user should add a Stimulus action for this behavior. This feature should come out of the box from the Stimulus Controller.

I would suggest initializing an event listener in connect (and tear it down in disconnect) to make it available out of the box.

Reply:
I like the idea of not having to maintain this in your HTML. The dropdown also has a `window` action that can probably be refactored in a similar fashion, so I think we can merge this and then refactor both at once.